### PR TITLE
Test for fix_warnings

### PR DESCRIPTION
**Pull request by kilotaras**
Added tests that verify that default implementation of fix_warnings correctly marks warnings as fixed/not_fixed depending on result of fix_line

#### ✔ Checklist:

- [x] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [ ] Tests for the changes have been added (for bug fixes / features);
- [ ] Docs have been added / updated (for bug fixes / features).

Unsure if checklist apply.

### CODE REVIEW

**Comment by mgrachev** [comment]
@kilotaras 👋 Thank you for your contribution! 🚀

Please add your PR to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md).

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### 🔧 Changed
 
+- Add tests for default implementation of Fix::fix_warnings ([@kilotaras](https://github.com/kilotaras))
```
Comment (mgrachev) [inline CHANGELOG.md:19]:
```suggestion
- Add tests for default implementation of Fix::fix_warnings [#266](https://github.com/dotenv-linter/dotenv-linter/pull/266) ([@kilotaras](https://github.com/kilotaras))
```

Comment (kilotaras) [reply CHANGELOG.md]: My bad for missing. Fixed.

**Comment by mgrachev** [comment]
@kilotaras Thank you! 👍

🙏 If it’s not difficult for you, please support the project - click on the star on GitHub ⭐️

**Comment by mgrachev** [comment]
@kilotaras Thank you! 👍

🙏 If it’s not difficult for you, please support the project - click on the star on GitHub ⭐️

**Pull request merged**